### ISSUE TITLE
Module download fails via Terraform registry

### ISSUE THREAD

**Issue by gusse** [nozaq/terraform-aws-lambda-auto-package#20]
It looks like the release tag has been set to "v0.3.0" while Terraform registry (and earlier versions) do not contain the "v" in the release versions.

Thus downloading the module fails:
```
│ Error: Failed to download module
│
│ Could not download module "lambda_example" (.terraform/modules/example/lambda.tf:1) source code from
│ "git::https://github.com/nozaq/terraform-aws-lambda-auto-package?ref=0.3.0": error downloading
│ 'https://github.com/nozaq/terraform-aws-lambda-auto-package?ref=0.3.0': /usr/local/bin/git exited with 1: error:
│ pathspec '0.3.0' did not match any file(s) known to git.
│ .
╵
```
This is my module configuration (as suggested by the example in registry):
```
module "lambda_example" {
  source  = "nozaq/lambda-auto-package/aws"
  version = "0.3.0"
...
}

**Comment by nozaq** [comment]
@gusse Ohhhh, thank you for reporting.

I just published v0.3.0. Other versions will also be published shortly.

**Comment by nozaq** [comment]
@gusse Thank you for reporting it! I just pushed `0.3.0` tag, could you try again?
